packages/discord.js/package.json

```diff
@@ -70,7 +70,7 @@
     "@discordjs/formatters": "workspace:^",
     "@discordjs/rest": "workspace:^",
     "@discordjs/util": "workspace:^",
-    "@discordjs/ws": "workspace:^",
+    "@discordjs/ws": "1.1.1",
```

Comment:
We can use a wider range

```suggestion
    "@discordjs/ws": "^1.1.1",
```

Reply:
Every other dependency is pinned without ranges, including `@discordjs/collection`. Don't see why this one should be different
cc @iCrawl